moesif-servlet/src/main/java/com/moesif/servlet/wrappers/BodyHandler.java

```diff
@@ -0,0 +1,28 @@
+package com.moesif.servlet.wrappers;
+
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+
+public class BodyHandler {
+
+  public static final int MAX_BODY_SIZE = 450 * 1024; // The Moesif API has a limit of 450KB for request and response body
```

Comment:
This should be configurable options `requestMaxBodySize` `responseMaxBodySize`, with the default set to 1MB. 1MB is what we support (not 450)

Reply:
Yeah ok I can add this to a configurable option.  I added this because in the ticket comments, we noted that 450kb is the collector limit

Reply:
Also update readme with these two options.

Reply:
@praves77 ok can do.  Max body size will be a new option. What is the second one you are referencing?

Reply:
@bakennedy No worries. Other SDK's have two different settings for request/response
1. Please update README with new options: https://github.com/Moesif/moesif-servlet/blob/master/README.md
2. https://github.com/Moesif/moesif-nodejs/blob/master/README.md#requestmaxbodysize here format is not markdown but we want to use markdown.